tests/framework/Optimizers/SimulatedAnnealing/BoltzmanEggHolder/test_simulatedAnnealing_eggHold.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="all">
+  <TestInfo>
+    <name>framework/Optimizers.SimulatedAnnealing</name>
+    <author>MohammadAbdo</author>
+    <created>2020-02-11</created>
+    <classesTested>SimulatedAnnealing</classesTested>
+    <description>
+      Extending on the Basic test, this test assesses the Simulated Annealing algorithm.
```

Comment:
can you expand the description?

Reply:
this comment is valid for all the other tests...

Reply:
Done.